tests/moduletool/test_install.py

```diff
@@ -1204,7 +1204,7 @@ def test_module_install_logging(local_module_package_index: str, snippetcompiler
         python_requires=v2_requirements,
         install_project=False,
         project_requires=[
-            module.InmantaModuleRequirement.parse("std==4.1.7"),
+            module.InmantaModuleRequirement.parse("std==4.2.1"),
```

Comment:
I haven't looked too much into this, but why not `4.3.3` ? This looks inconsistent with the previous behaviour ?

Reply:
How so, inconsistent? I mainly didn't want to pick the latest version because then we wouldn't really be testing anything.

Reply:
Inconsistent is maybe not the right word here, I mean the version previously used was `4.1.7` everywhere.
The version in some tests have been bumped from `4.1.7` to `4.2.1`  and to `4.3.3` in some others (e.g. `test_moduletool_list`).

> then we wouldn't really be testing anything.

How's so ?